install_config/cluster_metrics.adoc

```diff
@@ -64,39 +65,38 @@ address of the node, then Heapster will fail to retrieve any metrics.
 ====
 endif::[]
 
-The components for cluster metrics must be deployed to the `openshift-infra`
+The components for cluster metrics must be deployed to the *openshift-infra*
 project. This allows link:../dev_guide/pod_autoscaling.html[horizontal pod
-autoscalers] to discover the heapster service and use it to retrieve metrics
+autoscalers] to discover the Heapster service and use it to retrieve metrics
 that can be used for autoscaling.
 
 All of the following commands in this topic must be executed under the
-`openshift-infra` project. To switch to the `openshift-infra` project:
+*openshift-infra* project. To switch to the *openshift-infra* project:
 
-[options="nowrap"]
 ----
 $ oc project openshift-infra
 ----
 
-Next: To enable cluster metrics, configure the:
+To enable cluster metrics, you must next configure the following:
 
-. link:../install_config/cluster_metrics.html#service-accounts[Service Accounts]
-. link:../install_config/cluster_metrics.html#metric-data-storage[Metric Data Storage]
+. link:../install_config/cluster_metrics.html#metrics-service-accounts[Service Accounts]
+. link:../install_config/cluster_metrics.html#metrics-data-storage[Metrics Data Storage]
 . link:../install_config/cluster_metrics.html#metrics-deployer[Metrics Deployer]
```

Comment:
@adellape Does this have to be done in this order? If not, I'd suggest bullets instead of a numbered list.